src/util.cpp

```diff
@@ -801,11 +801,9 @@ int GetNumCores()
 
 std::string CopyrightHolders(const std::string& strPrefix)
 {
-    std::string strCopyrightHolders = strPrefix + _(COPYRIGHT_HOLDERS);
-    if (strCopyrightHolders.find("%s") != strCopyrightHolders.npos) {
-        strCopyrightHolders = strprintf(strCopyrightHolders, _(COPYRIGHT_HOLDERS_SUBSTITUTION));
-    }
-    if (strCopyrightHolders.find("Bitcoin Core developers") == strCopyrightHolders.npos) {
+    std::string strCopyrightHolders = strPrefix + strprintf(_(COPYRIGHT_HOLDERS), _(COPYRIGHT_HOLDERS_SUBSTITUTION));
```

Comment:
I don't think we support COPYRIGHT_HOLDERS with no "%s" in it. (If you really wanted to do that you'd also have to assert that COPYRIGHT_HOLDERS_SUBSTITUTION is empty? etc)

If someone feels like changing that, they can figure out a fix for themselves. There is no need to keep unused code in our repo because someday someone might or might not use it.